docs/source/reproducers/03-zuul.md

```diff
@@ -137,6 +145,35 @@ $ tail -f ~/ansible.log
 
 Note that `tmux` is installed by default, and may be handy in such a case ;).
 
+#### Zuul_inventory.yml
+That file is a generated inventory, allowing ansible to know about the compute(s) and CRC nodes.
+
+It exposes each node with the connection information (user and IP), and also created groups in
+order to access, for instance, all of the computes. Those groups mimic the same behaviour we
+have in Zuul.
+
+### Ansible tags of interest
+You may want to re-run some parts of the reproducer only, and avoid re-running the whole
+virtual machine creation.
+
+In order to do so, the reproducer exposes two tags:
+* bootstrap_layout
+* bootstrap
+
+#### Tag: bootstrap
```

Comment:
I'm not following the difference between `bootstrap` and `bootstrap_layout`.
IIUC, `bootstrap_layout` can be skipped, but `bootstrap` not?

Reply:
Both are needed (i.e. not --skip-tags) in order to get the deploy. Then, in case you want to iterate, you may skip them `bootstrap_layout` - this will jump over all of the VM bootstrap, while allowing to fetch and generate the various files of the PR you're re-running.
This is useful in case you're iterating a code change and see a failure in Zuul for each iteration - with that possibility to skip the VM bootstrap (`--skip-tags bootstrap_layout`) you may re-use the same env.
The `bootstrap` is a bit more aggressive, and would prevent most of the file generation, in addition to skipping most of the `layout` part (i.e. VM creation).

Both tags have an overlap, but don't cover exactly the same areas.

Reply:
reworded a bit, added some words of caution, and reorganized the tag list order.